assets/src/components/detoursTable.tsx

```diff
@@ -1,49 +1,98 @@
+      <DetourRows data={data} />
+    </tbody>
+  </Table>
+)
+
+const DetourRows = ({ data }: { data: SimpleDetour[] | null }) =>
+  data === null ? (
+    <EmptyDetourRows message="No draft detours." />
+  ) : (
+    <PopulatedDetourRows data={data} />
+  )
+
+const PopulatedDetourRows = ({ data }: { data: SimpleDetour[] }) => {
   const epochNowInSeconds = useCurrentTimeSeconds()
 
   return (
-    <Table hover className="c-detours-table">
-      <thead className="u-hide-for-mobile">
-        <tr>
-          <th className="px-3 py-4">Route and direction</th>
-          <th className="px-3 py-4 u-hide-for-mobile">Starting Intersection</th>
-          <th className="px-3 py-4 u-hide-for-mobile">On detour since</th>
-        </tr>
-      </thead>
-      <tbody>
-        {data.map((detour, index) => (
-          <tr key={index}>
-            <td className="align-middle p-3">
-              <div className="d-flex">
-                <RoutePill routeName={detour.route} />
-                <div className="c-detours-table__route-info-text d-inline-block">
-                  <div className="pb-1 fs-4 fw-semibold">{detour.name}</div>
-                  <div className="c-detours-table__route-info-direction fs-6">
-                    {detour.direction}
-                  </div>
+    <>
+      {data.map((detour, index) => (
+        <tr key={index}>
+          <td className="align-middle p-3">
+            <div className="d-flex">
+              <RoutePill routeName={detour.route} />
+              <div className="c-detours-table__route-info-text d-inline-block">
+                <div className="pb-1 fs-4 fw-semibold">{detour.name}</div>
+                <div className="c-detours-table__route-info-direction fs-6">
+                  {detour.direction}
                 </div>
               </div>
-            </td>
-            <td className="align-middle p-3 u-hide-for-mobile">
-              {detour.intersection}
-            </td>
-            <td className="align-middle p-3 u-hide-for-mobile">
-              {timeAgoLabel(epochNowInSeconds, detour.updatedAt)}
-            </td>
-          </tr>
-        ))}
-      </tbody>
-    </Table>
+            </div>
+          </td>
+          <td className="align-middle p-3 u-hide-for-mobile">
+            {detour.intersection}
+          </td>
+          <td className="align-middle p-3 u-hide-for-mobile">
+            {timeAgoLabel(epochNowInSeconds, detour.updatedAt)}
+          </td>
+        </tr>
+      ))}
+    </>
   )
 }
+
+const EmptyDetourRows = ({ message }: { message: string }) => (
+  <tr>
+    <td colSpan={3} className="p-3 p-md-4">
+      <div className="d-flex justify-content-center mb-3">
+        <EmptyDetourTableIcon height="100px" width="100px" />
+      </div>
+      <div className="d-flex justify-content-center">
+        <p className="fs-3 fw-light m-0">{message}</p>
+      </div>
+    </td>
+  </tr>
+)
+
+export const EmptyDetourTableIcon = (props: ComponentProps<"svg">) => (
+  <svg
```

Comment:
Ohhh... I read this and then forgot about it once I saw the code out of context. Hmm, perhaps I should reply there? But how do we avoid distributing otherwise-reusable icons throughout the repo in a way that is hard to track?

Reply:
Another worthwhile question: should this icon (and other icons) be moved somewhere more consistent? 

I figured it was likely that this icon would only ever be used on this page, so it wasn't necessary to create a separate home for it, but I don't feel strongly about that.

Reply:
maybe a `skateIcons.tsx` similar to `bsIcons.tsx`? Then we can have consistent tests for them. IMO This would be for "simple" icons which contain only SVG and don't take other props.

Reply:
That makes sense to me. Or perhaps a folder that contains a Reacty icon component per file? I prefer surfing icons by file name, but that's maybe too-personal a preference.

Also, turning my feedback to non-blocking, since this had already been discussed (and I forgot), and the requested change is mainly an organizational one at this point

Reply:
I like the `bsIcons` route because it makes it easy to apply uniform tests to all the icons without needing to edit the tests. But I'm open to other options.